gammagl/models/specformer.py

```diff
@@ -0,0 +1,298 @@
+import math
+from numpy.linalg import eigh  #  return eigenvalues and eigenvectors
+import numpy as np
+import tensorlayerx as tlx
+from tensorlayerx.model import WithLoss
+
+
+def feature_normalize(x):
+    x = np.array(x)
+    rowsum = x.sum(axis=1, keepdims=True)
+    rowsum = np.clip(rowsum, 1, 1e10)
+    return x / rowsum
+def normalize_graph(g):  # input : g is adj matrix , return the L matrix of graph

```

Comment:
I think these functions do not belong to this file, you may put these into the `example/specformer/utils` which means you need to create a new folder and put these functions into this folder.